specifyweb/frontend/js_src/lib/components/WbUtils/index.tsx

```diff
@@ -127,6 +129,28 @@ export function WbUtilsComponent({
         totalCount={cellCounts.newCells}
         utils={utils}
       />
+      {
+      // Only show these cells if batch-edit
+       isUpdate === true && <><Navigation
+        label={wbText.updatedCells()}
+        name="updatedCells"
+        totalCount={cellCounts.updatedCells}
+        utils={utils}
+      />
+      <Navigation
+        label={wbText.deletedCells()}
+        name="deletedCells"
+        totalCount={cellCounts.deletedCells}
+        utils={utils}
+      />
+      <Navigation
+        label={wbText.matchAndChanged()}
```

Comment:
it's not nice that `And` is capitalized in this button label

you can add `normal-case` class name to these buttons to prevent it from auto-capitalizing every word, thus giving you control over casing